### ISSUE TITLE
hosting policy reporter UI from a sub path

### ISSUE THREAD

**Issue by shilpa-0** [kyverno/policy-reporter#54]
Hi,

Is there any option to host the policy reporter UI from a sub path? 

Regards,
Shilpa

**Comment by fjogeleit** [comment]
hey, thank you for your request.

Do you mean something like "https://domain.com/policy-reporter-ui" ?

**Comment by shilpa-0** [comment]
@fjogeleit: yes 

For example, in kube-prometheus-stack helm chart, there is the option configure the Ingress resource for alertmanager:  
https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L210-L229

Could you provide a similar feature for the policy-reporter-ui?

**Comment by fjogeleit** [comment]
This feature already exists. It's in the UI Subchart

https://github.com/kyverno/policy-reporter/blob/main/charts/policy-reporter/charts/ui/values.yaml#L75-L89

So you can configure it in your policy-reporter values.yaml under the `ui` key.

```yaml
ui:
  ingress:
    enabled: true
    ...
```

**Comment by shilpa-0** [comment]
I somehow missed it :/ Thank you 👍

**Comment by shilpa-0** [comment]
Configuring ingress is not enough. I tried this out by configuring an ingress resource manually and it did not work.
Does the underlying UI application support being hosted on a sub-path? 
Could you please support this?

**Comment by shilpa-0** [comment]
Configuring ingress is not enough. I tried this out by configuring an ingress resource manually and it did not work.
Does the underlying UI application support being hosted on a sub-path?
Could you please support this?

**Comment by fjogeleit** [comment]
I will try to create an example today. What kind of ingress controller do you use?

**Comment by shilpa-0** [comment]
@fjogeleit: nginx. ingress-nginx to be specific: https://github.com/kubernetes/ingress-nginx

**Comment by fjogeleit** [comment]
I tried it by myself and I'm sorry but it's not that easy as it should. Because VueJS need the subPath at build time, so it is static defined in the docker image and I did not find a solution to configure it at runtime.

I will try to find a solution but it can take a few days.

**Comment by shilpa-0** [comment]
@fjogeleit: Thank you for the update

**Comment by fjogeleit** [comment]
@shilpa-0 I found a solution for this problem and created a new Policy Report UI version. I will publish a new helm release soon. If you want to try it out before you can use a similar configuration like this:

```yaml
ui:
  image: 
    tag: 0.13.2

  enabled: true

  plugins:
    kyverno: true

  ingress:
    enabled: true
    annotations:
      nginx.ingress.kubernetes.io/rewrite-target: /$1
    hosts:
      - host: domain.com
        paths:
        - "/policy-reporter/(.*)"
```

Important - the UI will be available under `http://domain.com/policy-reporter/` - the last `/` is required.

**Comment by fjogeleit** [comment]
The subPath support is released with 1.8.7 of the Helm Chart
